### ISSUE TITLE
Server world crash when upgrading existing world

### ISSUE THREAD

**Issue by mcdodge34** [TheLimePixel/Modern-Skyblock-3#312]
I tried it both in single player and in the server, here's the crash I get, just downloaded the server, copied the world on it and that's what I get, may I add that I'm far from Galacticraft stage, didn't discover anything yet in this direction, didn't open the quests or anything.

https://pastebin.com/6MM6zHeQ

**Comment by mcdodge34** [comment]
Keeps getting better and better, each time I try to start it, its a different crash report, there was one related to the Hats mod, I disabled it thinking this would solve the problem, now I get one that I have 0 clue what could happen there this time

https://pastebin.com/RQRfs5Cw

**Comment by mcdodge34** [comment]
I've manually fixed the issue by upgrading all the mods in the pack to their latest version and using foamfix lawful edition, I'M still unsure if I'll move my world to this version cause I had strange things like the questbook showing everything like if it was unlocked but its grey and doesn't provide clue at what to do to open the questline. Also the Openblocks elevators are gone from the world for some strange reason but I saw it was a config that disabled them, so not a big issue, I'll just modify the config for it. Also a complication seen in the Embers category, I had finished it and now there's a new complicated quest that requires the random failure of the damn alchemy table, I was fedup with that already.

Just curious if I have to do all the quests of the previous completed questlines to be ok with the rest of them or the stuff I've already unlocked will still be available.
